### PR TITLE
media-sound/whipper: remove dev-libs/gobject-introspection dependency and version bump to 0.7.2 and 0.7.3

### PR DESCRIPTION
**Pull request by CaseOf**
When I was writing the ebuild, I thought media-sound/whipper-0.7.0 was depending on dev-libs/gobject-introspection.
I tried to use it without dev-libs/gobject-introspection installed, and it works fine.
Then I deduce that media-sound/whipper-0.7.0 does not depend on it.
Also it is not listed in [README file](https://github.com/whipper-team/whipper/blob/v0.7.0/README.md#required-dependencies).

### CODE REVIEW

**Comment by CaseOf** [comment]
I added 0.7.2 ebuild too.

**Comment by CaseOf** [comment]
> I believe the first two commits are unnecessary. If you introduce 0.7.0-r1, 0.7.2 and 0.7.3 together with **same** keywords, everyone is going to end up installing 0.7.3 anyway, right? Just bump to 0.7.3 in 1st commit, then remove 0.7.0 in 2nd commit.

I did it this way in case there's an issue on newer versions, people would still be able to use a previous one that could not have it.

**Comment by juippis** [comment]
Right, remember to clean older versions though every now and then if there are no bug reports.

The EAPI-7 comment concers all ebuilds, then ;)

**Comment by CaseOf** [comment]
> The EAPI-7 comment concers all ebuilds, then ;)

Updated

File: media-sound/whipper/whipper-0.7.0-r1.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
```
Comment (juippis) [inline media-sound/whipper/whipper-0.7.0-r1.ebuild]: For the record, I'm not 100 % sure about the policy about `git mv`, you might get asked to split -r1 bump & removal of -r0 to two separate commits.

**Pull request closed without merging**